src/event_storage.js

```diff
@@ -65,23 +64,26 @@ class EventStorage {
         const collectEvents = () => {
             this._collectEvents(this._session.id())
         };
-        // Set up an interval to send evenst periodically
-        // TODO: Make sure using setInterval is not a terrible idea
-        this._interval = setInterval(collectEvents, EVENTS_PING_INTERVAL);
```

Comment:
Oops, shouldn't have removed this.

Reply:
Need to add a check before calling this functions though. QML does not have `setInterval` AFAIK.

Reply:
Ah, I see I put it below...